### ISSUE TITLE
if task['due_date_utc'] and is_habit(task['content']): KeyError: 'due_date_utc'

### ISSUE THREAD

**Issue by behicsakar** [amitness/habitist#9]
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.5/dist-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app.py", line 11, in main
    habits.main()
  File "/root/habitist/habits.py", line 46, in main
    if task['due_date_utc'] and is_habit(task['content']):
  File "/usr/local/lib/python3.5/dist-packages/todoist/models.py", line 17, in __getitem__
    return self.data[key]
KeyError: 'due_date_utc'
```

**Comment by behicsakar** [comment]
When i print the task object i found an error that is
```
  File "app.py", line 11, in main
    habits.main()
  File "/root/habitist/habits.py", line 47, in main
    print(task)
UnicodeEncodeError: 'ascii' codec can't encode character '\U0001f195' in position 97: ordinal not in range(128)
```

When i research about the char '\U0001f195' its 🆕. I think it's about the todoist API

**Comment by behicsakar** [comment]
I am not sure but i created new issue in the todoist-python in Doist. If im wrong please correct me.
https://github.com/Doist/todoist-python/issues/49

**Comment by amitness** [comment]
Looks like they fixed it in API version 8.0. I've upgraded the library and made the changes.
